.github/workflows/ci.yml

```diff
@@ -36,17 +36,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: p0deje/setup-bazel@0.1.0
+      - run: bazel test ...
+        working-directory: .
       - run: echo 'RUBY_VERSION = "${{ matrix.ruby }}"' > ruby_version.bzl
       - run: bazel build ...
       - run: bazel run lib/gem:add-numbers 2
       - run: bazel run lib/gem:print-version
       - run: bazel test ...
       - if: failure() && runner.debug == '1'
         uses: mxschmitt/action-tmate@v3
```

Comment:
```suggestion
        uses: mxschmitt/action-tmate@v3
        
  ruleset:
    name: Ruleset
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - run: bazel test ...
```

Reply:
Yeah this is fine for now as we only care to run the rules_ruby workspace tests on a single platform 👍🏻